conf/routes

```diff
@@ -19,7 +19,7 @@ GET    /v3/subpopulations        @org.sagebionetworks.bridge.play.controllers.Su
 POST   /v3/subpopulations        @org.sagebionetworks.bridge.play.controllers.SubpopulationController.createSubpopulation
 GET    /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.getSubpopulation(guid: String)
 POST   /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.updateSubpopulation(guid: String)
-DELETE /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.deleteSubpopulation(guid: String)
+DELETE /v3/subpopulations/:guid  @org.sagebionetworks.bridge.play.controllers.SubpopulationController.deleteSubpopulation(guid: String, physical: String ?= "false")
```

Comment:
See note above. From a REST resource perspective, the URL is just the URL to delete this thing. Considerations like security, content negotiation, language, etc. should not be differentiated by URL.